src/ProjectTemplates/Web.ProjectTemplates/content/Worker-CSharp/Program.Main.cs

```diff
@@ -4,13 +4,10 @@ public class Program
 {
     public static void Main(string[] args)
     {
-        IHost host = Host.CreateDefaultBuilder(args)
-            .ConfigureServices(services =>
-            {
-                services.AddHostedService<Worker>();
-            })
-            .Build();
+        HostApplicationBuilder builder = Host.CreateApplicationBuilder(args);
+        builder.Services.AddHostedService<Worker>();
 
+        IHost host = builder.Build();
```

Comment:
`var` all the things, what is this!? 😉

Reply:
I was conflicted because the existing template doesn't use `var`, it has `IHost`. How strongly do we feel about this in templates?

Reply:
I'd make it match what we have in the web template